src/views/v2/month/mobile-events/mobile-day.php

```diff
@@ -52,16 +53,24 @@
 
 <div <?php tribe_classes( $classes ); ?> id="<?php echo sanitize_html_class( $mobile_day_id ); ?>">
 
-	<?php foreach ( $events as $event ) : ?>
-		<?php $event_date = $event->dates->start->format( Dates::DBDATEFORMAT ); ?>
+	<?php if ( count($events) ) : ?>
 
-		<?php $this->template( 'month/mobile-events/mobile-day/day-marker', [ 'events' => $events, 'event' => $event, 'request_date' => $day_date ] ); ?>
+		<?php foreach ( $events as $event ) : ?>
+			<?php $event_date = $event->dates->start->format( Dates::DBDATEFORMAT ); ?>
 
-		<?php $this->setup_postdata( $event ); ?>
+			<?php $this->template( 'month/mobile-events/mobile-day/day-marker', [ 'events' => $events, 'event' => $event, 'request_date' => $day_date ] ); ?>
 
-		<?php $this->template( 'month/mobile-events/mobile-day/mobile-event', [ 'event' => $event ] ); ?>
+			<?php $this->setup_postdata( $event ); ?>
 
-	<?php endforeach; ?>
+			<?php $this->template( 'month/mobile-events/mobile-day/mobile-event', [ 'event' => $event ] ); ?>
 
-	<?php $this->template( 'month/mobile-events/mobile-day/more-events', [ 'more_events' => $day['more_events'], 'more_url' => $day['day_url'] ] ); ?>
+		<?php endforeach; ?>
+
+		<?php $this->template( 'month/mobile-events/mobile-day/more-events', [ 'more_events' => $day['more_events'], 'more_url' => $day['day_url'] ] ); ?>
+
+	<?php else : ?>
+
+		<?php $this->template( 'components/messages', [ 'attributes' => [ 'aria-hidden' => 'true' ], 'messages' => $mobile_messages ] ); ?>
```

Comment:
we can probably remove `aria-hidden` here, since we don't actually change this in the JS, it hurts us more than not.